ingress/controllers/gce/controller/utils.go

```diff
@@ -188,7 +188,10 @@ type StoreToIngressLister struct {
 // List lists all Ingress' in the store.
 func (s *StoreToIngressLister) List() (ing extensions.IngressList, err error) {
 	for _, m := range s.Store.List() {
-		ing.Items = append(ing.Items, *(m.(*extensions.Ingress)))
+		newIng := m.(*extensions.Ingress)
+		if isGCEIngress(newIng) {
```

Comment:
Would we need this check (to exclude GCE ingress controllers) in the nginx controller too ? Or there are no resources wasted so we don't need it there ?